ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ReplLoadWork.java

```diff
@@ -61,12 +63,19 @@
   */
   final LineageState sessionStateLineageState;
 
-  public ReplLoadWork(HiveConf hiveConf, String dumpDirectory, String dbNameToLoadIn,
-      LineageState lineageState, boolean isIncrementalDump, Long eventTo,
-      List<DirCopyWork> pathsToCopyIterator) throws IOException {
+  public ReplLoadWork(HiveConf hiveConf, String dumpDirectory,
+                      String dbNameToLoadIn, ReplScope changedReplScope,
```

Comment:
the input is not changed repl scope ..its the original repl scope ..

Reply:
It is in fact changed/new repl policy. To be clear, I will rename it to currentReplPolicy. Hope that is fine.